src/main/java/games/strategy/engine/data/Resource.java

```diff
@@ -1,8 +1,13 @@
 package games.strategy.engine.data;
 
+import java.util.ArrayList;
+import java.util.List;
+
 public class Resource extends NamedAttachable {
   private static final long serialVersionUID = 7471431759007499935L;
 
+  private List<PlayerID> players = new ArrayList<>();
```

Comment:
I think this field can now be `final` if you remove the assignment.

Reply:
Good point. Updated.